vms/avm/vm_test.go

```diff
@@ -218,10 +226,19 @@ func BuildGenesisTest(t *testing.T) []byte {
 	}
 
 	return hex.Bytes
+
```

Comment:
super minor nit here, but I don't think anywhere else in the repo do we have code blocks that start or end with whitespace. (I think this shows up in a few of the functions here)